### DISCUSSION TITLE
Download as zip

### DISCUSSION THREAD

**Discussion by AbidFaiez** [coder/code-server#4834] [General]
While downloading project with code-server got some problems ( big number of files) so it crush . so I want to change the download behavior to download as zip or tar like Theia eclipse do.
Is it possible ??

**Comment by jsjoeio** [comment]
Hmm... not that I'm aware of. Does this exist in VS Code?

**Reply by AbidFaiez** [reply]
Nope but exists in Theia eclipse

**Reply by jsjoeio** [reply]
Well unfortunately then it's not available in code-server. We are essentially a fork of VS Code

**Reply by AbidFaiez** [reply]
Thank you
